openmls/src/treesync/diff.rs

```diff
@@ -83,13 +83,28 @@ impl<'a> TryFrom<&'a TreeSync> for TreeSyncDiff<'a> {
 }
 
 impl<'a> TreeSyncDiff<'a> {
-    /// Check if the right-most leaf is blank. If that is the case, remove the
-    /// right-most leaf until the right-most leaf is not blank anymore.
+    /// Check if the right-most leaf and its parent are blank. If that is the
```

Comment:
Please add something here that the code here differs from the spec and that this is an ongoing discussion.